ui/src/app/utils/access-utils.tsx

```diff
@@ -422,13 +437,22 @@ export const computeRenewalDisplayDates = ({
   const nextReviewDate = withInvalidDateHandling(expirationEpochMillis);
   const bypassDate = withInvalidDateHandling(bypassEpochMillis);
 
+  function currentOrExpiringOrExpirationStatus(): AccessModulesStatus {
```

Comment:
can shorten the name

also, please add a comment about which modules are valid for this call: only those which are complete